.github/workflows/Commitlint.yml

```
@@ -5,7 +5,7 @@ on:
     branches: [main]
 
 jobs:
-  build:
+  check-commit-message:
```

nit, but would be nice to stick with a consistent naming convention, so:
```suggestion
  Check-Commit-Message:
```